pangeo_forge_recipes/aggregation.py

```diff
@@ -287,13 +287,19 @@ def schema_to_zarr(
     attrs: Optional[Dict[str, str]] = None,
     consolidated_metadata: Optional[bool] = True,
     encoding: Optional[Dict] = None,
+    append_dim: Optional[str] = None,
```

Comment:
Is it worth adding some validation to see if this append_dim (str) is a valid dataset dimension?

Reply:
Yes! I do that in StoreToZarr.__post_init__. Also, xarray.Dataset.to_zarr does that internally, I believe.